spec/memory-safe-d.dd

```diff
@@ -24,19 +24,53 @@ $(HEADERNAV_TOC)
 
 $(H2 $(LNAME2 usage, Usage))
 
-        $(P Memory safety can be enabled on a per-function basis using
-        the $(DDSUBLINK spec/function, safe-functions, $(D @safe) attribute).
-        This can be inferred when the compiler has the function body
-        available. The $(DDSUBLINK spec/function, trusted-functions, `@trusted` attribute) can be used when a function has a safe
-        interface, but uses unsafe code internally. These functions can
-        be called from $(D @safe) code.
+        $(P There are three categories of functions from the perspective of memory safety:)
+        $(UL
+                $(LI $(DDSUBLINK spec/function, safe-functions, $(D @safe)) functions)
+                $(LI $(DDSUBLINK spec/function, trusted-functions, $(D `@trusted`)) functions)
+                $(LI $(DDSUBLINK spec/function, system-functions, $(D `@system`)) functions)
         )
 
+        $(P $(D `@system`) functions may perform any operation legal from the perspective of the language including inherently
+        memory unsafe operations like returning pointers to expired stackframes. These functions may not be called directly from
+        $(D `@safe`) functions.)
+
+        $(P $(D `@trusted`) functions have all the capabilities of $(D `@system`) functions but may be called from
```

Comment:
Why do you double escape things here? Markdown backticks do the same as the D Ddoc macro.